lib/prebundler/s3_backend.rb

```diff
@@ -33,15 +40,15 @@ def list_files
       files = []
       base_options = {
         bucket: bucket,
-        prefix: "#{Bundler.local_platform.to_s}/#{Prebundler.platform_version}/#{Gem.extension_api_version.to_s}"
+        prefix: "#{Bundler.local_platform}/#{Prebundler.platform_version}/#{Gem.extension_api_version}"
       }
 
       while truncated
         options = if continuation_token
-          base_options.merge(continuation_token: continuation_token)
-        else
-          base_options
-        end
+                    base_options.merge(continuation_token: continuation_token)
```

Comment:
I know this is Lumos style, but I personally much prefer only a single level of indentation :)

Reply:
damn editors....